### DISCUSSION TITLE
Need to hit enter twice?

### DISCUSSION THREAD

**Discussion by jbhouse** [Aloxaf/fzf-tab#332] [Q&A]
$ cd *hit tab*

*select my directory from the fzf completion dropdown*

$ cd dirname *I now have to hit enter AGAIN before hitting tab again to look through subdirectories under dirname*

Hopefully this is a decent explanation, as I cannot google this for the life of me

But why do I have to hit *enter* after selecting something from fzf-tab, before I can type, hit tab again, etc

**Comment by Aloxaf** [comment]
Can you provide your zshrc?

**Reply by jbhouse** [reply]
turns out this issue was due to zsh-autosuggestions
I thought the issue was fzf-tab related because I am the guy that reported the 6 commits back bug, and therefore assumed this issue would be related

Sorry for the confusion here
